tests/validation/tests/v3_api/common.py

```diff
@@ -2650,40 +2650,53 @@ def delete_resource_in_AWS_by_prefix(resource_prefix):
         'Values': [resource_prefix + "-*"]
     }]
     nodes = AmazonWebServices().get_nodes(filters=node_filter)
+
+    # Check the r53 record to see if the value match a node ip address
+    # this indicates we're using an nginx load balancer
+
+    rancher_ha_hostname = resource_prefix + ".qa.rancher.space."
+    r53_record = AmazonWebServices().get_route_53_record(rancher_ha_hostname)
+    r53_record_ip = r53_record['ResourceRecords'][0]['Value']
+    nginx_node = [node for node in nodes if r53_record_ip == node.public_ip_address]
```

Comment:
Instead of relying on comparison of IP addresses, can we add a tag to the node on creation?